1-installing-bolt/README.md

```diff
@@ -4,72 +4,175 @@
 
-## CentOS 7/RHEL 7
+**Note:** These packages require you to download the Puppet 5 Platform. To install only the Bolt package you can install the packages directly as well.
 
-```
-yum install -y make gcc ruby-devel
-gem install bolt
-```
+1.  Download and install the software and its dependencies. Use the commands appropriate to your system.
+    -   Debian 8
 
-## Fedora 25
+        ```
+        wget https://apt.puppet.com/puppet5-release-jessie.deb
+        sudo dpkg -i puppet5-release-jessie.deb
+        sudo apt-get update 
+        sudo apt-get install bolt
+        
+        ```
+
+    -   Debian 9
+
+        ```
+        wget https://apt.puppet.com/puppet5-release-stretch.deb
+        sudo dpkg -i puppet5-release-stretch.deb
+        sudo apt-get update 
+        sudo apt-get install bolt
+        ```
+
+    -   Ubuntu 14.04
+
+        ```
+        wget https://apt.puppet.com/puppet5-release-trusty.deb
+        sudo dpkg -i puppet5-release-trusty.deb
+        sudo apt-get update 
+        sudo apt-get install bolt
+        ```
+
+    -   Ubuntu 16.04
+
+        ```
+        wget https://apt.puppet.com/puppet5-release-xenial.deb
+        sudo dpkg -i puppet5-release-xenial.deb
+        sudo apt-get update 
+        sudo apt-get install bolt
+        ```
+
+2.  Run a Bolt command to verify it has installed correctly. 
+
+    ```
+    bolt --help
+    ```
+
+
+## Install Bolt from yum repositories on RHEL or SLES
+
+Packaged versions of Bolt are available for Red Hat Enterprise Linux 6 and 7, SUSE Linux Enterprise Server 12.
+
+The Puppet repository for the YUM package management system is [http://yum.puppet.com/puppet5/](http://yum.puppet.com/puppet5/) Packages are named using the convention `<PLATFORM_NAME>-release-<OS ABBREVIATION>-<OS VERSION>.noarch.rpm`. For example, the release package for Puppet 5 Platform on Linux 7 is `puppet5-release-el-7.noarch.rpm`.
+
+**Note:** These packages require you to download the Puppet 5 Platform. To install only the Bolt package you can install the packages directly as well.
+
+1.  Download and install the software and its dependencies. Use the commands appropriate to your system.
+    -   Enterprise Linux 6
+
+        ```
+        sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-el-6.noarch.rpm
+        sudo yum install bolt				
+        ```
+
+    -   Enterprise Linux 7
+
+        ```
+        sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-el-7.noarch.rpm
+        sudo yum install bolt
+        ```
+
+    -   SUSE Linux Enterprise Server 12
+
+        ```
+        sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-sles-12.noarch.rpm
+        sudo yum install bolt
+        ```
+
+2.  Run a Bolt command  to verify it has installed correctly.  
+
+    ```
+    bolt --help
+    ```
+
+
+## Install Bolt as a gem
+
+Install Ruby 2.3 or above and Bolt.
+
+If Ruby is already included in your operating system, verify that it is version 2.3 or above.
```

Comment:
Can we include `ruby --version` there, just so folks don't have to google how to find the ruby version? Pretty straightforward, but I think we also assume inexperienced users.

Reply:
Added.